storio-sqlite/src/test/java/com/pushtorefresh/storio2/sqlite/integration/NotifyAboutChangesTest.java

```diff
@@ -197,11 +203,12 @@ public void run() {
     }
 
     @Test
+    @Repeat(times = 20)
```

Comment:
I've seen this test failed on #827 PR build, but I don't think I ever saw it fail before, so I just added `@Repeat` to make sure it's good, I've read it carefully and still think it's a good test as is :)